package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@teamleader/ui-illustrations",
   "description": "Teamleader UI Illustrations",
-  "version": "1.19.1",
+  "version": "1.20",
```

Comment:
Can you add the `v` to the version, since the tags are already like that too.

Reply:
Tags need to be created through CI now, which https://github.com/teamleadercrm/ui-visuals/pull/401 will facilitate

Reply:
Oh ok that explains why I'm having difficulties with the tags.

Reply:
That is not a valid semver version though: https://docs.npmjs.com/cli/v10/configuring-npm/package-json